app/Http/Resources/FailedJobResource.php

```diff
@@ -15,11 +16,14 @@ class FailedJobResource extends JsonResource
      */
     public function toArray($request)
     {
-        $channelSubscribe = null;
```

Comment:
Così facendo, se io non vado dentro l'if di riga 21, poi alle righe 34 e 35 mi ritrovo che `$channelSubscribe` non è stata definita.

Reply:
Che però dovrebbe essere gestito correttamente dal ternario. Se vuoi cambio comunque

Reply:
In realtà nel ternario la condizione verifica solo il valore truthy o falsy di `$channelSubscribe`, ma non controlla se la variabile è stata definita oppure no. E per come è stato modificato, si dà il caso che la variabile non esista.